liota/dcc_comms/mqtt_dcc_comms.py

```diff
@@ -74,18 +74,19 @@ def __init__(self, edge_system_name, url, port, identity=None, tls_conf=None, qo
 
         self.client_id = client_id
 
+        if self.client_id is None:
+            #  local_uuid generated will be the client ID
+            self.client_id = systemUUID().get_uuid(edge_system_name)
+            log.debug("Auto-Generated local uuid will be the client ID")
+        else:
+            log.debug("Client ID is provided by user")
+
         if mqtt_msg_attr is None:
             #  pub-topic and sub-topic will be auto-generated
-            log.info("pub-topic and sub-topic is auto-generated")
+            log.debug("Pub-topic and Sub-topic are auto-generated")
```

Comment:
s/are/will be/ ? (We could/should log them?)

Reply:
It is logged in MqttMessagingAttributes